internal/server/volume/internal/v1alpha1/conversion_generated.go

```diff
@@ -128,25 +128,13 @@ func Convert_internal_IsVolumeFormattedResponse_To_v1alpha1_IsVolumeFormattedRes
 }
 
 func autoConvert_v1alpha1_ListVolumesOnDiskRequest_To_internal_ListVolumesOnDiskRequest(in *v1alpha1.ListVolumesOnDiskRequest, out *internal.ListVolumesOnDiskRequest) error {
```

Comment:
right, it looks like the autoConvert functions will always be generated, I think that making changes to these _generated.go files is error prone because at some point we might delete them and want to auto generate them (at least that's the workflow that I usually follow) and the manual changes will be gone.

The easiest approach is to just ignore the fact that they're unused, we could document that they're unused and that would be it.

Another approach would be to modify the generator to stop generating these `auto` functions when it detects that there's a conversion function for it.